packages/containers/psammead-timestamp-container/src/index.test.jsx

```diff
@@ -63,4 +65,40 @@ describe('Timestamp', () => {
       service="news"
     />,
   );
+
+  describe('assertions', () => {
+    it('should render a hausa timestamp', () => {
+      const { container } = render(
+        <Timestamp
+          timestamp={defaultTimestamp}
+          dateTimeFormat="YYYY-MM-DD"
+          format="D MMMM YYYY"
+          isRelative={false}
+          script={latin}
+          locale="ha"
+          service="hausa"
+        />,
+      );
+
+      const time = container.querySelector('time');
+      expect(time.textContent).toEqual('19 Oktoba 2018');
+    });
+
+    it('should render a hausa relative timestamp', () => {
+      const { container } = render(
+        <Timestamp
+          timestamp={defaultTimestamp}
+          dateTimeFormat="YYYY-MM-DD"
+          format="D MMMM YYYY"
+          isRelative
+          script={latin}
+          locale="ha"
+          service="hausa"
+        />,
+      );
+
+      const time = container.querySelector('time');
+      expect(time.textContent).toEqual('11 months wuce');
```

Comment:
Is this going to break in a months time?

Reply:
What do you suggest to test a relative time then?

Reply:
Would the following help?
```
moment(Date.now())
          .subtract(11, 'months')
          .unix();
```

Reply:
Will try that, thanks!